### PR TITLE
Adds support for Danish language

### PR DESCRIPTION
**Pull request by PJ789**
@facchinm please could you merge into Keyboard?

(.This pull requests supersedes previous pull request)

### CODE REVIEW

**Comment by PJ789** [comment]
@edgar-bonet - this is the same code you reviewed previously, with the change to fix the backslash scan code. Apologies for any confusion.

**Review by edgar-bonet — APPROVED** [review]
Other than a typo in my name in the commit log, it all looks good to me.

File: src/Keyboard.cpp [hunk 1]
```diff
@@ -1,214 +1,588 @@
 /*
-  Keyboard.cpp
+    Syonara - Arduino SS Micro (Leonardo) firmware for Razer Cynosa Lite and similar
```
Comment (edgar-bonet) [inline src/Keyboard.cpp:2]: Did you push here some code that was actually intended for your Syonara repository?

Comment (PJ789) [reply src/Keyboard.cpp]: Yes, sorry... I've fixed that. So embarrassed.

Comment (PJ789) [reply src/Keyboard.cpp]: PS. Syonara is cool. It takes a perfectly serviceable keyboard with seriously dodgy OEM firmware, and replaces it with a new ATMega32u4 running the keyboard library (using the keyboard status led/HID pull requests to control the LEDs).

**Comment by facchinm** [comment]
Squashed and merged, thanks!

**Pull request merged**